gcpvault_test.go

```diff
@@ -44,6 +45,7 @@ func TestGetSecrets(t *testing.T) {
 				Role:       "my-gcp-role",
 				LocalToken: "my-local-token",
 				SecretPath: "my-secret-path",
+				MaxRetries: 1,
```

Comment:
Why did you have to set all the tests to MaxRetries==1 ?

Reply:
I wanted to make sure that tests that fail would fail fast. But I see it's not necessary.